cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -1239,6 +1274,14 @@ end
 ###############################################################################
 
 policy "pol_databricks_optimizations" do
+  validate $ds_verified_tags do
+    summary_template "Azure Databricks ClusterId Tag Dimension Not Configured in Flexera."
+    detail_template "The **Azure Databricks ClusterId** tag dimension is required to properly allocate costs by mapping each Azure Virtual Machine to its associated Databricks Cluster. This tag dimension is currently missing in your Flexera configuration.
```

Comment:
For multi-line strings like this, EOS should be used, like so:

```ruby
  detail_template <<-EOS
The **Azure Databricks ClusterId** tag dimension is required to properly allocate costs by mapping each Azure Virtual Machine to its associated Databricks Cluster. This tag dimension is currently missing in your Flexera configuration.

To resolve this, create the required custom tag dimension in Flexera by following the documentation: [**Create a Custom Tag Dimension in Flexera**](https://docs.flexera.com/flexera/EN/Optima/CreatingCustomTags.htm)
EOS
````

Reply:
Still slightly off. Added a suggestion here - https://github.com/flexera-public/policy_templates/pull/3210#discussion_r2194432323